src/sandbox/sandbox.go

```diff
@@ -0,0 +1,74 @@
+package sandbox
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"syscall"
+
+	// #include "sandbox.h"
+	"C"
+)
+
+const MS_LAZYTIME = 1 << 25
+
+func Sandbox(args []string) error {
+	cmd := exec.Command(args[0], args[1:]...)
+
+	cmd.Stdout = os.Stdout
+	cmd.Stdin = os.Stdin
+	cmd.Stderr = os.Stderr
+	cmd.Dir = "/tmp/plz_sandbox"
+	if err := mountTmp(); err != nil {
+		return err
+	}
+	if i := C.lo_up(); i < 0 {
+		return fmt.Errorf("failed to bring loopback interface up")
+	}
+
+	fmt.Println("Welcome to the sandbox!")
+	return cmd.Run()
+}
+
+func mountTmp() error {
+	tmpDir := os.Getenv("TMP_DIR")
+	dir := "/tmp/plz_sandbox"
```

Comment:
this should probably use `core.SandboxDir` - or that should use this - until we can unify it all in here.

Reply:
I was thinking the benefit of doing something like this would be to make it parse os.Args() or do you think that would be too slow?